requirements-dev.txt

```diff
@@ -10,5 +10,6 @@ moto[server]
 pytest
 pytest-cov
 pytest-xdist
+python-dateutil<2.7 # required for botocore=1.9.8
```

Comment:
I'm trying to figure out why it's also necessary to add this under requirements-dev. Does one of our other dev requirements upgrade python-dateutil?

Reply:
It might not matter.  It depends on the order in which people install things.  For example, it is necessary (I think, might need verification) if someone does the following:
```
pip install -U -e .
pip install -U -r requirements-dev.txt
```

In any case, the *correct* place for the pinning is in the dev requirements because girder itself doesn't even depend on boto.  It just happens tox does the installation in the opposite order.

Reply:
Girder does depend on botocore, but indirectly. It's a requirement of boto3.

Reply:
Right.  It's an indirect dependency, which is primarily the reason why pip isn't correctly resolving the `python-dateutil<2.7` restriction correctly.

Reply:
Ok, I was just a bit confused by having a non-dev requirement listed in requirements-dev, but I just wasn't thinking about the way we use requirements-dev.txt. It actually does make sense.